Source/WebCore/Modules/mediastream/gstreamer/GStreamerWebRTCUtils.cpp

```diff
@@ -491,7 +491,7 @@ uint32_t UniqueSSRCGenerator::generateSSRC()
 
 std::optional<int> payloadTypeForEncodingName(StringView encodingName)
 {
-    static UncheckedKeyHashMap<String, int> staticPayloadTypes = {
+    static HashMap<String, int> staticPayloadTypes = {
```

Comment:
This should really be using `SortedArrayMap` instead. Maybe there are others like that.